hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenMintHandler.java

```diff
@@ -16,18 +16,37 @@
 
 package com.hedera.node.app.service.token.impl.handlers;
 
-import static com.hedera.hapi.node.base.ResponseCodeEnum.INVALID_TOKEN_ID;
+import static com.hedera.hapi.node.base.ResponseCodeEnum.*;
```

Comment:
can you configure your IntelliJ to not do * imports?

<img width="1094" alt="imports" src="https://github.com/hashgraph/hedera-services/assets/9443847/37eb8415-687c-4f3a-a4bf-8739445f463f">

Reply:
Done. Thanks